src/components/ConfirmDialogWindow.jsx

```diff
@@ -0,0 +1,72 @@
+import { useA11yDialog } from 'react-a11y-dialog';
+import { createPortal } from 'react-dom';
+import './ConfirmDialogWindow.css';
+
+const ConfirmDialogWindow = ({ text, title, confirmAction }) => {
+	// `instance` is the `a11y-dialog` instance.
+	// `attr` is an object with the following keys:
+	// - `container`: the dialog container
+	// - `overlay`: the dialog overlay (sometimes called backdrop)
+	// - `dialog`: the actual dialog box
+	// - `title`: the dialog mandatory title
+	// - `closeButton`:  the dialog close button
+	const [instance, attr] = useA11yDialog({
+		id: 'confirm-dialog',
+		role: 'dialog',
+		title: 'Confirm action dialog',
+	});
+
+	const dialog = createPortal(
```

Comment:
Nice use of a portal. This is a perfect use case for using React [Portals](https://reactjs.org/docs/portals.html)!

Reply:
I wish it was mine invention, but this part comes from the creator of the a11y dialog, I used the example for a custom dialog provided in the documentation